extension/docs/HowToRun.md

```diff
@@ -88,23 +88,34 @@ Multiple child attributes can be provided in the `commercetools` attribute. Each
+      }
     }
   }
 }
 ```
 
-| Name           | Content                                                      | Required | Default value                                     |
-| -------------- | ------------------------------------------------------------ | -------- | ------------------------------------------------- |
-| `clientId`     | OAuth 2.0 `client_id` and can be used to obtain a token.     | YES      |                                                   |
-| `clientSecret` | OAuth 2.0 `client_secret` and can be used to obtain a token. | YES      |                                                   |
-| `host`         | The commercetools HTTP API is hosted at that URL.            | NO       | `https://api.europe-west1.gcp.commercetools.com`  |
-| `authUrl`      | The commercetools’ OAuth 2.0 service is hosted at that URL.  | NO       | `https://auth.europe-west1.gcp.commercetools.com` |
+| Name            | Content                                                                                                                                                                                                            | Required | Default value                                     |
+| --------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | -------- | ------------------------------------------------- |
+| `clientId`      | OAuth 2.0 `client_id` and can be used to obtain a token.                                                                                                                                                           | YES      |                                                   |
+| `clientSecret`  | OAuth 2.0 `client_secret` and can be used to obtain a token.                                                                                                                                                       | YES      |                                                   |
+| `host`          | The commercetools HTTP API is hosted at that URL.                                                                                                                                                                  | NO       | `https://api.europe-west1.gcp.commercetools.com`  |
+| `authUrl`       | The commercetools’ OAuth 2.0 service is hosted at that URL.                                                                                                                                                        | NO       | `https://auth.europe-west1.gcp.commercetools.com` |
+| `authetication` | Enable authentication mechanism to prevent unauthorized access. If it is provided, it must contains `scheme` attribute which supports `basic` type, `username` attribute and `password` attribute defined by user. | NO       |                                                   |
```

Comment:
```suggestion
| `authetication` | Enable authentication mechanism to prevent unauthorized access to the extension module. If it is provided, it must contains `scheme` attribute which supports `basic` type, `username` attribute and `password` attribute defined by user. | NO       |                                                   |
```

Reply:
Also add a sentence that you need to have `basicAuth` enabled for this option to work.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/584/commits/367a5cafdd2c30934b0975de92bfe4631a68d243